Error: 
  â”Œ tests/js_error/source_locations/expression/super_member/get_named_super_property.js:3:11
  |
3 |     throw new Error();
  |           ^
Stack:
  at get foo (tests/js_error/source_locations/expression/super_member/get_named_super_property.js:3:11)
  at test (tests/js_error/source_locations/expression/super_member/get_named_super_property.js:9:10)
  at <global> (tests/js_error/source_locations/expression/super_member/get_named_super_property.js:14:1)